code/modules/mining/equipment/survival_pod.dm

```diff
@@ -310,3 +310,6 @@
 	desc = initial(I.desc)
 	icon_state = initial(I.icon_state)
 	inhand_icon_state = initial(I.inhand_icon_state)
+	lefthand_file = initial(I.lefthand_file)
+	righthand_file = initial(I.righthand_file)
+	cut_overlays() //to get rid of the big blue x
```

Comment:
why do you need to cut_overlays here? what big blue x gets applied to the item?

Reply:
If you spawn it in, then right click on it, or attack someone with it, you'll see what I mean. There's a big blue x overlay on it when it's spawned. May be because I'm tired but I can't figure out a better way to do this.